composer.json

```
@@ -13,7 +13,7 @@
       }
    ],
    "require": {
-      "php": ">=7.3.0"
+      "php": ">=7.4.0"
```

If you're trying to test on PHP 8, you'll need to specify that here: 
```suggestion
      "php": "^7.3|^8.0"
```

---

Any good reason to deprecate 7.3 here? Is it incompatible? Matyroshka is public afterall.

---

If we want to start using property types I think we'll have to deprecate 7.3, but we aren't doing that currently. I actually just did it cause Travis has started limiting free build minutes, and I thought it might be good to conserve resources :shrug:

I've fine with leaving 7.3 in there for now since it is still compatible, and we haven't run up against any build limits yet.

---

Where did you see that syntax with the `|` for PHP versions? Installing the package on PHP 8 with the current syntax isn't warning/erroring, so I'm not sure how to validate.

---

Okay, I peeked at some other `composer` examples, and I think I found the right syntax. Pushed a change that addresses your suggestions: 1a4a87e